lala1

```diff
@@ -2,3 +2,4 @@ hi, I am lala
 I am 12 years old
 I am in grade 4
 I like to draw
+But I don't like to color.
```

Comment:
why this line?

Reply:
Because I don't have colors, only pencil